### PR TITLE
Example of exception testing in spock

### PR DESCRIPTION
**Pull request by tkurylek**

### CODE REVIEW

File: src/test/groovy/com/gmaslowski/spock/sample/GroovyDummyServiceTest.groovy [hunk 1]
```diff
@@ -45,4 +46,16 @@ class GroovyDummyServiceTest extends Specification {
         then:
         2 * dummyRepository.getDummyById(dummyId)
     }
+
+    def "should not deactivate dummy with negative id"() {
```
Comment (gmaslowski) [inline src/test/groovy/com/gmaslowski/spock/sample/GroovyDummyServiceTest.groovy:50]: The aim of the project was to show differences between java and groovy + spock testing. Could you a java example?

Comment (gmaslowski) [inline src/test/groovy/com/gmaslowski/spock/sample/GroovyDummyServiceTest.groovy:50]: Could you also extract the test to other file? I tried to keep the sample tests files as small and descriptive as possible.

Comment (tkurylek) [inline src/test/groovy/com/gmaslowski/spock/sample/GroovyDummyServiceTest.groovy:50]: Sure thing, I will do it as soon as I have some spare time.

Comment (gmaslowski) [inline src/test/groovy/com/gmaslowski/spock/sample/GroovyDummyServiceTest.groovy:50]: Actually I can do it as well :). Thanks for the job :).

**Pull request merged**